fbpic/fields/fields.py

```diff
@@ -777,10 +777,10 @@ def divide_by_volume( self, fieldtype ) :
         for m in range(self.Nm):
             self.interp[m].divide_by_volume( fieldtype )
 
-    def divide_by_volume_and_e0( self, fieldtype ) :
+    def divide_by_volume_envelope( self, fieldtype ) :
```

Comment:
I added the division by epsilon directly in the GPU kernel. 
This is because dividing a device array by a scalar (in pure Python) - as is done here - is not supported.